components/HeaderAppBar/index.tsx

```diff
@@ -0,0 +1,84 @@
+// Things to fix: the color of the app bar is blue, but idk how to change that to make it white like
+// the figma example
+
+import * as React from 'react'
+import AppBar from '@mui/material/AppBar'
+import Box from '@mui/material/Box'
+import Toolbar from '@mui/material/Toolbar'
+import IconButton from '@mui/material/IconButton'
+import Typography from '@mui/material/Typography'
+import Menu from '@mui/material/Menu'
+import MenuIcon from '@mui/icons-material/Menu'
+import Container from '@mui/material/Container'
+import Avatar from '@mui/material/Avatar'
+import MenuItem from '@mui/material/MenuItem'
+import { blue } from '@mui/material/colors'
+
+const settings = ['Sign out']
+
+interface HeaderAppBarProps {
+  setDrawerOpen: (status: boolean) => void
+}
+
+export default function HeaderAppBar(props: HeaderAppBarProps) {
+  const [anchorElUser, setAnchorElUser] = React.useState<null | HTMLElement>(
+    null
+  )
+
+  const handleOpenUserMenu = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorElUser(event.currentTarget)
+  }
+
+  const handleCloseUserMenu = () => {
+    setAnchorElUser(null)
+  }
+
+  return (
+    <AppBar position="static" color="transparent" sx={{ px: 2 }}>
+      <Toolbar disableGutters>
+        <Box sx={{ flexGrow: 1, display: { xs: 'flex', md: 'none' } }}>
+          <IconButton
+            size="large"
+            aria-label="account of current user"
+            aria-controls="menu-appbar"
+            aria-haspopup="true"
+            color="inherit"
+            onClick={() => props.setDrawerOpen(true)}
+          >
+            <MenuIcon />
+          </IconButton>
+        </Box>
+        {/* keeps the sign in user circle in the right most corner */}
+        <Box sx={{ flexGrow: 1, display: { xs: 'none', md: 'flex' } }}></Box>
+
+        <Box sx={{ flexGrow: 0 }}>
+          <IconButton onClick={handleOpenUserMenu} sx={{ p: 0 }}>
+            <Avatar alt="Remy Sharp" src="/static/images/avatar/2.jpg" />
+          </IconButton>
+          <Menu
+            sx={{ mt: '45px' }}
+            id="menu-appbar"
+            anchorEl={anchorElUser}
+            anchorOrigin={{
+              vertical: 'top',
+              horizontal: 'right',
+            }}
+            keepMounted
+            transformOrigin={{
+              vertical: 'top',
+              horizontal: 'right',
+            }}
+            open={Boolean(anchorElUser)}
+            onClose={handleCloseUserMenu}
+          >
+            {settings.map((setting) => (
```

Comment:
As it stands there will only ever be one option here, so do we need this map? @zaviermiller Do we want to keep it here in case we want to expand its functionality? I can't think of anything else we'd put there